source/imports/domain/bank-accounts/bank-account.js

```diff
@@ -0,0 +1,44 @@
+import Contact from '../../value-objects/contact';
+import BankAccountOpened from '../../domain-events/bank-account-opened';
+import _ from 'lodash';
+
+const BankAccount = Space.eventSourcing.Aggregate.extend('BankAccount', {
+  
+  fields: {
+    balance: Money,
+    owner: Contact
+  },
+
+  commandMap() {
+    return {
+      'OpenBankAccount': this._openBankAccount
```

Comment:
Maybe use the real class type here, as it adds to the understanding of the code (where commands and events come from) `[OpenBankAccount]: this._openBankAccount`